types/types.go

```diff
@@ -147,7 +147,7 @@ type NodeConfig struct {
 }
 
 type HostRequirements struct {
-	SSE3         bool `json:"sse3,omitempty"`          // sse3 cpu instruction
+	SSSE3        bool `json:"sse3,omitempty"`          // ssse3 cpu instruction
```

Comment:
json attribute left as 'sse3' - on purpose?

Reply:
nope. needs to be fixed